scripts/plutus/staking-example/register-stake-pool.sh

```diff
@@ -130,7 +130,7 @@ cardano-cli transaction build \
   --testnet-magic "$TESTNET_MAGIC" \
   --change-address "$utxoaddrwithstaking" \
   --tx-in "$keytxin" \
-  --tx-out "$utxoaddrwithstaking+1000" \
```

Comment:
What's the reason for these changes?

Reply:
Minimum utxo requirement. I changed the config files to mimic mainnet a few weeks back but did not update all the scripts.